code/general/membership.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Membership.NewMapSetMember do
+  def check_membership(range, list, _mapset) do
+    find = Enum.random(range)
```

Comment:
I'm fairly certain that the runtime of `Enum.random/1` should be constant, but for the sake of making these comparisons even stronger, how would you feel about moving this call to a [`before_each` hook](https://hexdocs.pm/benchee/readme.html#hooks-setup-teardown-etc) so the runtime of `Enum.find/1` isn't included in the comparison?